README.md

```diff
@@ -8,42 +8,35 @@ Monitor your Heimdall, Bor and Virtual machine node with Prometheus and Grafana
 
 **Step 1:**
 
+By default these changes should have already been made as part of your node setup but incase they're not, you can follow the steps below
+
 You will need to change `prometheus` flag to `true` in the `config.toml`. 
 
 To access the config.toml here:
 
-**Linux Packages**: `sudo vi /etc/heimdall/config/config.toml`
-**Binaries**: `vi ~/.heimdalld/config/config.toml`
-
-By default the prometheus flag is set to `false`.
+`vi ~/.heimdalld/config/config.toml`
 
 Make sure that you keep the formatting intact.
 
 **Step 2:**
 
-Once you have made the changes in the `config.toml` you will need to stop Heimdall service and restart it.
+Once you have made the changes in the `config.toml` you will need to stop Heimdall service and restart it. Note that you only need to restart Heimdall service if you have made changes to it.
 
 To stop heimdall service:
 
-**Linux Packages**: `sudo service heimdalld stop`
-**Binaries**: `pkill heimdalld`
+`sudo service heimdalld stop`
 
 To restart heimdall service:
 
-**Linux Packages**: `sudo service heimdalld start`
-**Binaries**: `heimdalld start`
+`sudo service heimdalld start`
 
 **Step 3:**
 
 You will make changes in the Bor service file. To access the `bor.service` file you can run this command:
 
-`locate bor.service`
-
-There would be multiple entries. You will need to edit the file in this system path: `/etc/systemd/system/bor.service`
-
-To edit the service file you can run the following command:
+`nano /home/ubuntu/node/bor/start.sh`
 
-`sudo nano /etc/systemd/system/bor.service`
+There would be multiple entries. By default the following parameters should be automatically added, but if they're not added then you can the parameters from below.
 
 Now in this you would see `ExecStart=/bin/bash` with multiple paramaters in line to it. Add this, `--metrics --pprof --pprofport 7071 --pprofaddr 0.0.0.0` to this line of paramaters. You can add it anywhere, for example, you can add it after `--maxpeers 150`
```

Comment:
@delroybosco this not clear I guess, pls mention the bor start.sh file path.

Reply:
Yes. Needs to be in `~/node/bor/start.sh`